lib/datadog/opentelemetry/extensions.rb

```diff
@@ -2,7 +2,9 @@
 require 'datadog/tracing/span_operation'
 require 'datadog/opentelemetry/span'
 
+# Datadog
 module Datadog
```

Comment:
Minor: 🤔  this comment doesn't seem very useful?

Reply:
Agreed but Rubocop is dumb.